src/types/message.ts

```diff
@@ -18,38 +18,14 @@ interface MessageBase {
    */
   type: string;
 }
-
-// Definitions for all messages from GCS to vehicles.
-
+// Definitions for all messages from GCS to vehicles
```

Comment:
still showing changes here if you look at file changes, you dont need any file changes here so to undo it just add an empty line on top and bottom of the comment